Week2/homework/Masoud/gradeCalculator/gradeCalculator.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+// creating function to change score or point to grade...
+function gradeCalculator(point) {
+  let grade;
+  switch(true) {
+    case (point >= 0 && point < 49):
+      grade = 'F';
+      break;
+    case (point >= 50 && point < 59):
+      grade = 'E';
+      break;
+    case (point >= 60 && point < 69):
+      grade = 'D';
+      break;
+    case (point >= 70 && point < 79):
+      grade = 'C';
+      break;
+    case (point >= 80 && point < 89):
+      grade = 'B';
+      break;
+    case (point >= 90 && point <= 100):
+      grade = 'A';
+      break;
+  }
+return grade;
```

Comment:
Requirement:
Make the return value of the function a template string, so you can insert variables!